frontends/p4/sideEffects.h

```diff
@@ -42,6 +42,13 @@ class HasTableApply : public Inspector {
         setName("HasTableApply");
     }
 
+    explicit HasTableApply(TypeMap *typeMap)
+        : refMap(this), typeMap(typeMap), table(nullptr), call(nullptr) {
+        CHECK_NULL(refMap);
+        CHECK_NULL(typeMap);
+        setName("HasTableApply");
+    }
```

Comment:
```suggestion
    explicit HasTableApply(TypeMap *typeMap) : HasTableApply(this, typeMap) {}
```
... I believe this should work too

Reply:
Yes, it was just quick copy & paste :)